demo/package.json

```
@@ -1,47 +1,35 @@
 {
   "nativescript": {
-    "id": "org.nativescript.demo",
+    "id": "nl.essent.urbanairship",
```

is this supposed to be in there?
```suggestion
    "id": "org.nativescript.demo",
```